src/components/NotificationsTest.tsx

```diff
@@ -70,6 +70,15 @@ export default function NotificationTest({ permissions }: Props) {
     }
   };
 
+  const unregisterPushNotifications = async () => {
+    try {
+      await PushNotifications.register();
```

Comment:
Just noticed you changed unregister to register.
The PR should have failed to build until a new release of the plugin was available, but it passed because of the change.